src/MarkerLineChart.js

```diff
@@ -225,7 +237,8 @@ export default class MarkerLineChart extends React.Component {
     if (!_.every([x1, x2, y1, y2], _.isFinite)) return null;
     return (
       <line
-        className="marker-line"
+        className={`${getValue(lineClassName, d, i)}`}
```

Comment:
does this need to be in strings?

Reply:
not sure what you mean by this but to clarify `getValue` takes as its first argument a value or accessor function. so if lineClassName is a value (a string) then it's returned, if it's a function then it's the result of that function applied with the rest of the arguments